### DISCUSSION TITLE
Updated .gitignore for V9+

### DISCUSSION THREAD

**Discussion by idseefeld** [umbraco/Umbraco-CMS#10786] [Features and ideas]
Hi,
I have recently updated https://github.com/github/gitignore regarding Umbraco. I know it might be to early for a final 
 version, but certain rules seem to be clear. 
There is already a discussion ongoing and I have the feeling this needs some _official_ support. In the end I do not know how they handle PRs. Do you know how this was handled in the past?

You'll find my pull request her: https://github.com/github/gitignore/pull/3780#issuecomment-891799938

Dirk

**Comment by nul800sebastiaan** [comment]
Thanks @idseefeld - it's community-maintained, so whatever you all think is best!
